threedigrid_builder/base/lines.py

```diff
@@ -38,8 +38,10 @@ class Line:
     frict_type2: int
     frict_value1: float  # For 1D-2Dgw: resistance factor-out (hydr. cond / thickness)
     frict_value2: float  # For 1D-2Dgw: resistance factor-in (hydr. cond / thickness)
-    veg_coef1: float  # the product of vegetation properties
-    veg_coef2: float  # the product of vegetation properties
+    veg_coef1: float  # the product of vegetation properties (except height)
+    veg_coef2: float  # the product of vegetation properties (except height)
+    veg_hght1: float  # the vegetaion height
```

Comment:
I think I would just write out height, we also have `cross_section_height` and `cross_weight`